kubernetes/kserve/developer_guide.md

```diff
@@ -7,8 +7,8 @@ The documentation covers the steps to run Torchserve along with the KServe for t
 Below are the prerequisites should be met.
 
 - Torchserve >= 0.6.0
-- Torch-model-archiver >= 0.6.0 
-- Kserve >= 0.8.0
+- Torch-model-archiver >= 0.6.0
```

Comment:
We just made a release for 0.6.1 so we can just add `pip install torchserve torch-model-archiver` etc.. instead of having to change this every release